tools/crm_resource_runtime.c

```diff
@@ -2208,9 +2209,9 @@ cli_resource_move(const pe_resource_t *rsc, const char *rsc_id,
     if (force && !cur_is_dest) {
         /* Ban the original location if possible */
         if(current) {
-            (void)cli_resource_ban(out, rsc_id, current->details->uname, move_lifetime,
-                                   NULL, cib, cib_options, promoted_role_only);
-
+            rc = cli_resource_ban(out, rsc_id, current->details->uname, move_lifetime,
+                                   NULL, cib, cib_options, promoted_role_only,
+                                   PCMK__ROLE_PROMOTED);
```

Comment:
The idea behind ignoring the return code is that the preference before this is sufficient, so it's OK if this fails